### PR TITLE
:memo: Moving large .NET projects rules

### PR DESCRIPTION
**Pull request by fenix2222**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

SSW Rules migration

> 2. What was changed?

Migrated Large .NET Projects Rule category

https://github.com/SSWConsulting/SSW.Rules.Content/issues/8313

### CODE REVIEW

**Comment by tiagov8** [comment]
@igorgoldobin 

FYI I am waiting for answer on other PRs before approve this

- https://github.com/SSWConsulting/SSW.Rules.Content/pull/8311
- https://github.com/SSWConsulting/SSW.Rules.Content/pull/8310
- https://github.com/SSWConsulting/SSW.Rules.Content/pull/8309

File: categories/software-engineering/rules-to-better-large-builds-in-visual-studionet.md
Comment (JackDevAU) [inline categories/software-engineering/rules-to-better-large-builds-in-visual-studionet.md:3]: @bradystroud @jaydenalchin

Can this category be archived?

**Review by JackDevAU — CHANGES_REQUESTED** [review]
see comment

File: categories/software-engineering/rules-to-better-large-builds-in-visual-studionet.md
Comment (jaydenalchin) [reply categories/software-engineering/rules-to-better-large-builds-in-visual-studionet.md:3]: I don't know enough on the topic to confirm or deny @JackDevAU

**Comment by tiagov8** [comment]
Hey @igorgoldobin 

I think you need to update this after https://github.com/SSWConsulting/SSW.Rules.Content/pull/8361

**Review by tiagov8 — APPROVED** [review]
Hi @igorgoldobin 

- [ ] 1. Please review all these rules live and make changes it needed.

I was not 100% about them, but approved anyways

**Pull request merged**
